conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/JaxRsClient.java

```diff
@@ -55,4 +64,23 @@ private JaxRsClient() {}
                         config,
                         serviceConfiguration -> create(serviceClass, userAgent, hostEventsSink, serviceConfiguration)));
     }
+
+    /**
+     * Creates a {@code T client} for the given dialogue {@link Channel}.
+     */
+    public static <T> T create(Class<T> serviceClass, ConjureRuntime runtime, Channel channel) {
```

Comment:
This is the only new public API.

Note that ConjureRuntime and Channel are from dialogue-target which is intentionally an `implementation` dependency to prevent leaking access to dialogue to consumers which don't already have it.

Reply:
`JaxRsClient.class` is relatively specific, I've added a test to show that this case fails with `IllegalStateException: Method <method name> not annotated with HTTP method type (ex. GET, POST)`